src/tools/genpolicy/src/registry.rs

```diff
@@ -133,28 +163,87 @@ impl Container {
         debug!("Getting process field from docker config layer...");
         let docker_config = &self.config_layer.config;
 
+        // Defaults to start with if we can't work out the correct values.
```

Comment:
Unless you are seeing test failures, let's remove these defaults - because the intended design was that the caller provides the default values.